packages/node_modules/@ciscospark/widget-recents/src/enhancers/setup.js

```diff
@@ -131,22 +131,30 @@ function getInitialSpaces(props) {
   if (!widgetStatus.isFetchingInitialSpaces
     && !widgetStatus.hasFetchedInitialSpaces) {
     props.updateWidgetStatus({isFetchingInitialSpaces: true});
-    let fetchCommand;
-
     if (props.basicMode) {
-      fetchCommand = props.fetchSpacesHydra(sparkInstance, {
+      props.fetchSpacesHydra(sparkInstance, {
         max: props.spaceLoadCount
-      });
+      })
+        .then(() => props.updateWidgetStatus({
+          hasFetchedInitialSpaces: true,
+          // Basic mode doesn't have a second load state
+          hasFetchedAllSpaces: true
+        }));
     }
     else {
       /**
        * Fetches an encrypted small batch of spaces
        * This allows us to show the encrypted spaces in the spaces list
        * and decrypt individually to give a good initial user experience.
        */
-      fetchCommand = props.fetchSpacesEncrypted(sparkInstance, {
+      props.fetchSpacesEncrypted(sparkInstance, {
         conversationsLimit: props.spaceLoadCount
       })
+        .then(() => props.updateWidgetStatus({
```

Comment:
Moving this before decryption brings back the encrypted spaces view that got hidden!